compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -35,6 +35,8 @@
 #define DBGSNL() (llvm::dbgs() << "\n")
 
 static constexpr unsigned kShuffleBitWidth = 32;
+static const char kGPUTargetAttrName[] = "iree_codegen.target_info";
+static const char kWavesPerEuAttrName[] = "waves_per_eu";
```

Comment:
This should be under something like `AMDGPUUtils.cpp`. Can you add a TODO to move it?